src/evict.c

```diff
@@ -708,3 +708,16 @@ int performEvictions(void) {
     return result;
 }
 
+int performEvictions(void) {
+    int result = doPerformEvictions();
+
+    if (result == EVICT_RUNNING) {
+        if (server.stat_last_eviction_exceeded_time == 0) {
+            server.stat_last_eviction_exceeded_time = ustime();
```

Comment:
Is it possible to use ```server.ustime```.

Reply:
I intend to use `server.ustime`. This function may be called in `serverCron`, the cached `server.ustime` may be outdated.